sdk/python/pulumi_aws_native/databrew/recipe.py

```diff
@@ -118,9 +118,9 @@ def __init__(__self__,
                 "action": {
                     "operation": "EXTRACT_PATTERN",
                     "parameters": {
-                        "sourceColumn": "Consulate",
+                        "source_column": "Consulate",
```

Comment:
This is a little suspect. I wonder if the change is right or it is a regression.